packages/driver/cypress.json

```diff
@@ -13,5 +13,6 @@
   "retries": {
     "runMode": 2,
     "openMode": 0
-  }
+  },
+  "experimentalMultiDomain": true
```

Comment:
Should we also be setting `experimentalSessionSupport` here?

Reply:
@mschile sorry I missed this comment! I don't think we need to because it likely would impact the `cookie_spec`. We can get away with setting `experimentalSessionSupport` as an override because the server doesn't need the value, which is nice!